web/styles/globals.css

```diff
@@ -0,0 +1,40 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    -webkit-tap-highlight-color: transparent;
```

Comment:
Are we sure we want to turn this off? When I tap a contract card on android, the blue highlight gives me some feedback about what I did.

Reply:
I feel like the feedback should be that the card's dropshadow gets bigger. I just think the blue highlight is ugly in general and we should just make better focus states to give feedback. I was wondering if there were maybe some accessibility considerations I wasn't thinking about though